### PR TITLE
temporary fix dockerfile start and env setup

### PR DESCRIPTION
**Pull request by stuart-bass-bjss**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -6,12 +6,15 @@ WORKDIR /usr/src/app
 # Install app dependencies
 COPY package*.json ./
 
-# RUN npm install
 RUN npm install
+#RUN npm ci --only=production
```
Comment (alibombaye) [inline Dockerfile]: Is this comment deliberate?

**Comment by stuart-bass-bjss** [comment]
Yes. The tech debt ticket has details to look at using one vs the other. I wanted to leave a marker in the file to remind us.

File: Dockerfile [hunk 2]
```diff
@@ -6,12 +6,15 @@ WORKDIR /usr/src/app
...
 COPY . .
 
 # Expose port 
-EXPOSE 3001
-CMD [ "node", "server.js", "npm run start" ]
+EXPOSE 3002
+# Run app
+#CMD [ "node", "server.js"]
+CMD [ "npm", "start"]
+#USER node
```
Comment (alibombaye) [inline Dockerfile]: This comment here. Is this prep for when we get it right?

**Comment by stuart-bass-bjss** [comment]
Yes, prep for getting it right.

**Pull request merged**